src/components/book.jsx

```diff
@@ -22,7 +22,9 @@ function Book({
             </button>
 
             <button
-              onClick={() => dispatch(removeBook(id))}
+              onClick={() => {
+                dispatch(deleteBook(item_id));
+              }}
               type="button"
```

Comment:
As previously pointed out by the previous reviewer, you app needs to be manually reloaded after a book is deleted and this is not a good user experience. I experimented with your code and realised that the better way of solving this is not working because of some bugs somewhere that requires much time to track, so I came up with alittle bit of shabby approach to fix this issue.

consider replating the highted line with this:
```
<button
              onClick={() => {
                dispatch(deleteBook(item_id));
                setTimeout(()=> {
                  window.location.reload();
                }, 100)
              }}
```

Adding this will resolve the issue and provide a better UX